dependencies.list

```diff
@@ -1,7 +1,7 @@
-PACKAGE_NAME=realm-core
```

Comment:
I'd suggest changing the file name to `dependencies.yaml` to formalize the change in syntax. 
This would have the effect of self documenting the expected syntax of the file, as well as intentionally breaking scripts or SDKs that depend on this which otherwise might have their parsers broken in difficult to discover ways. Either way, this change probably warrants a note in the "internals" section of the changelog.